src/target/adiv5.c

```diff
@@ -1072,6 +1072,41 @@ void firmware_mem_write_sized(adiv5_access_port_s *ap, uint32_t dest, const void
 	adiv5_dp_read(ap->dp, ADIV5_DP_RDBUFF);
 }
 
+void mm32l0_firmware_mem_write_sized(adiv5_access_port_s *ap, uint32_t dest, const void *src, size_t len, align_e align)
```

Comment:
This function should be in stm32f1.c as it's not ADIv5-generic, and drop the `_firmware_` - the naming of several of the functions in this file needs rework as they're wrong and that is part of how. (`firmware_mem_read`, for example, should be `adiv5_mem_read`).